src/main/java/nextstep/ladder/domain/Prizes.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.ladder.domain;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Prizes implements Iterable<Prize> {
+    public static final String DILIMETER = ",";
+    private final List<Prize> prizes;
+
+    private Prizes(List<Prize> prizes) {
+        this.prizes = prizes;
+    }
+
+    public static Prizes of(String names) {
+        return new Prizes(Arrays.stream(names.split(DILIMETER))
+            .map(Prize::of)
+            .collect(Collectors.toList()));
+    }
```

Comment:
```suggestion
    private Prizes(List<Prize> prizes) {
        this.prizes = prizes;
    }

    public static Prizes of(List<Prize> prizes) {
        return new Prizes(prizes);
    }

    public static Prizes of(String names) {
        return Prizes.of(Arrays.stream(names.split(DILIMETER))
            .map(Prize::of)
            .collect(Collectors.toList()));
    }
```
이렇게 기본 생성자를 위한 of도 하나 만들어서 관리해주면 어떨까요?